unstructured/partition/lang.py

```diff
@@ -221,7 +221,14 @@ def detect_languages(
     Detects the list of languages present in the text (in the default "auto" mode),
     or formats and passes through the user inputted document languages if provided.
     """
-    if text.strip() == "":
+    # Skip language detection for partitioners that use other partitioners.
+    # For example, partition_msg relies on partition_html and partition_text, but the metadata
+    # gets overwritten after elements have been returned by _html and _text,
+    # so `languages` would be detected twice.
+    if languages[0] == "":
+        return [""]
+
+    if text.strip() == "" or len(text) < 20:
```

Comment:
@shreyanid
I added this because "Stanley Cups" wasn't being detected as English when partitioning xlsx, but now one of the language tests (test_detect_languages_korean_auto) is failing because the text is only 5 characters long (though very obviously not English). Any suggestions on how to find a balance?

Reply:
Never mind. Thought of using regex to check if characters were all ascii

Reply:
@shreyanid This is the part I was referring to in the Ingest Test Fixtures Update PR.

Reply:
Where is the Ingest Test Fixtures Update PR? Yes, I saw the addition of regex to check for ascii characters, did that solve the short korean text problem?

Reply:
It's here:
https://github.com/Unstructured-IO/unstructured/pull/1652#event-10567029937

Yes, the regex solved the korean and Stanley Cups xlsx problem, but it looks like we have some other existing documents with short text that's not having the language detected accurately

Reply:
Ok, feel free to show some of those docs/results to me. Should we potentially default to English if these inaccurate short text documents can be identified?

Reply:
That's what I was trying to do with the regex pattern. It makes sure there are only English characters in the text and text is less than 20 characters so that if there's short text in English, `langdetect` won't incorrectly detect some other language. But it's not able to catch everything.

Reply:
It might be worth pushing this off to a follow-up issue where we also look more closely at other language detection packages and come up with our own tests to compare them all.

Reply:
That sounds good!

Reply:
https://github.com/Unstructured-IO/unstructured/issues/1660